fieldservice_activity/views/fsm_template.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_fsm_template_form_add_activity" model="ir.ui.view">
+        <field name="name">fsm.template.form.activity</field>
+        <field name="model">fsm.template</field>
+        <field name="inherit_id" ref="fieldservice.fsm_template_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[1]" position="after">
```

Comment:
https://travis-ci.org/OCA/field-service/jobs/631380913#L2084

I think the error is because we don't have a notebook in base fieldservice on the template form

Reply:
@brian10048 You are correct, I will be opening a new PR with changes in the fieldservice module that include that change (to keep template consistent with the form view of orders) and a 'Manage FSM Activites' checkbox in Settings.

Reply:
Sounds good to me